system/ee/ExpressionEngine/Library/Captcha.php

```diff
@@ -48,7 +48,7 @@ public function create($old_word = '', $force_word = false)
             $score_threshold = ee()->config->item('recaptcha_score_threshhold');
             $action_id = ee()->functions->fetch_action_id('member', 'recaptcha_check');
 
-            $js = "<script> var csrf_token='".CSRF_TOKEN."'; var endpoint ='".ee()->functions->fetch_site_index(0, 0) . QUERY_MARKER . 'ACT=' . $action_id."'; var key ='".$key."'; </script>";
+            $js = "var endpoint ='".ee()->functions->fetch_site_index(0, 0) . QUERY_MARKER . 'ACT=' . $action_id."'; var key ='".$key."'; </script>";
```

Comment:
Thinking of these further, should we use something less generic as variable names here? Or add namespace

Reply:
@intoeetive Fixed on latest commit.